manifests/nodejs.yml

```diff
@@ -786,6 +786,7 @@ tests/:
     test_stats.py:
       Test_Client_Stats: missing_feature
   test_config_consistency.py:
+    Test_Config_ClientIPHeaderEnabled_False: *ref_5_27_0
```

Comment:
this should work way befor 5.27.0 no ? i coded that feature years ago

Reply:
@simon-id Yes it should work before. I simply put latest release version for all languages for the sake of the OKR goal of writing the tests and enabling it for the languages that work.

Reply:
If you really want to do half the work, at least put a comment saying something like `# real version not known`

Reply:
@simon-id Hey, thanks for the feedback! I can address this in a followup PR.

Reply:
@simon-id That comment is unacceptable especially in a public forum.

Reply:
Oh, I thought I was talking to you this whole time. Sorry @mhlidd I took you for someone else :facepalm:

I've chatted with Ayan to try to solve this recurring issue on a systemic level instead of relying on culture. I couldn't find a good idea as this kind of mistake cannot realistically be caught by a CI for example.
Best thing we can rely on for now is a bit more clarity on how this manifest file is supposed to be used and maintained. Through more docs and more training maybe? I'll look into making some changes to the docs. Ultimately it's not your fault as your team was just not given the proper training. I'm happy to chat to find solutions to make this smoother.

Reply:
And thank you for the follow up PR btw :+1: